### PR TITLE
Add a simple sUSD faucet

### PR DESCRIPTION
**Pull request by masonforest**

### CODE REVIEW

**Review by karlfloersch — APPROVED** [review]
Nice!

The two things I'd do before shipping is 1) not commit the `package-lock.json` because it's just so massive, and 2) un-delete the `package.json` because that must have been added by accident.

Anyway, other than that, looks good!

File: package.json [hunk 1]
```diff
@@ -1,108 +0,0 @@
-{
-	"name": "synthetix",
-	"version": "2.20.0",
```
Comment (karlfloersch) [inline package.json]: Whoops! I think this was removed by accident

Comment (karlfloersch) [reply package.json]: And the `package-lock.json` added by accident

**Review by K-Ho — APPROVED** [review]
lgtm

**Pull request merged**